schema/transactions.js

```diff
@@ -214,8 +214,14 @@ module.exports = {
 			multisigAccountPublicKey: {
 				type: 'string',
 				format: 'publicKey'
+			},
+			data: {
+				type: 'string',
+				minLength: 0,
+				maxLength: 16
 			}
```

Comment:
What kind of format would we check against? Isn't data optional, and can hold any type of string value?